scripts/zones/Mine_Shaft_2716/mobs/Bugbby.lua

```diff
@@ -0,0 +1,72 @@
+-----------------------------------
+-- Area: Mine Shaft 2716
+-- CoP Mission 5-3 (A Century of Hardship)
+-- NM: Bugbby
+-----------------------------------
+require("scripts/globals/status")
+mixins = {require("scripts/mixins/job_special")}
+local ID = require("scripts/zones/Mine_Shaft_2716/IDs")
+require("scripts/globals/status")
+-----------------------------------
+local entity = {}
+
+local aCenturyOfHardship =
+{
+    [1] =
```

Comment:
Any raw ID should be referenced from the IDs.lua file (enum, and pull them into tables here if needed, that way if there are shifts, its contained to a subset of files)